packages/@react-aria/focus/test/FocusScope.test.js

```diff
@@ -779,6 +779,42 @@ describe('FocusScope', function () {
       expect(document.activeElement).toBe(item2);
     });
 
+    it('should move focus forward and allow users to skip certain elements', function () {
```

Comment:
I assume if both focusNext() and focusPrevious() work we don't need to test focusFirst() or focusLast().

Reply:
technically those could be useful, i didn't do it more so in the interest of time, we know that the focusableTreeWalker is doing the right thing with the argument, so these tests are just here to prevent the removal of that option, which I think if you're doing it to one of these four, you're likely doing it to all four of them, so I'm not worried about it